airflow/providers/pagerduty/provider.yaml

```diff
@@ -37,6 +37,13 @@ integrations:
     tags: [service]
 
 
+hook-class-names: # deprecated - to be removed after providers add dependency on Airflow 2.2.0+
```

Comment:
2.2.0 is going to be the next version, so this comment does not sem to make a lot of sense. Why is it here?

Reply:
I have looked at the code of other providers and this comment is present in most of their `provider.yaml` files, as such I have added it. E.g.
- airflow/providers/microsoft/azure/provider.yaml
- airflow/providers/jira/provider.yaml
- airflow/providers/slack/provider.yaml

I will remove it, if you deem it unnecessary.